content/applications/sales/sales/send_quotations/different_addresses.rst

```diff
@@ -2,74 +2,54 @@
-
-.. note::
-   If you select a customer with defined invoice and delivery addresses, Odoo will automatically use
-   them to fill in the fields. Now, if you want to change it instantly, it is possible to do so
-   directly from the quotation or the sales order.
+   :alt: Add a contact/address to the contact form
 
-Add addresses from a contact form
-=================================
+Then, select which type of address to add to the contact form (i.e. :guilabel:`Invoice Address` or
+:guilabel:`Delivery Address`).
 
-Previously, we talked about the contact form that you can fill in directly from a quotation or a
-sales order to add billing and shipping addresses to customers. But, if you go to
-:menuselection:`Sales --> Orders --> Customers`, you can create or modify every customer you want
-and add, delete or modify invoice and delivery addresses instantly there, before creating a
-quotation.
-
-.. image:: different_addresses/addresses_3.png
+.. image:: different_addresses/create-contact-window.png
    :align: center
    :class: img-thumbnail
-   :alt: How to add addresses from a contact form on Odoo Sales?
-
-Deal with different addresses
-=============================
+   :alt: Create a new contact/address on a contact form
 
-Like for the previous example, go to :menuselection:`Sales --> Orders --> Customers` and create a
-new customer. There, you can add company information but, more importantly, you can enter billing
-and shipping addresses under the **Contacts & Addresses** tab.
+Enter the address information. Then, click :guilabel:`Save & Close` to save the address and close
+the :guilabel:`Create Contact` window, or click :guilabel:`Save & New` to save this address and
+immediately input another one.
 
-.. image:: different_addresses/addresses_4.png
-   :align: center
-   :class: img-thumbnail
-   :alt: How to manage different customer addresses on Odoo Sales?
+Add addresses to the quotation
+==============================
 
-Once done, you can return to your Sales dashboard and create a new quotation. Now, if you enter
-your new customer, you will see that the other fields will fill in by themself with the information
-previously saved for billing and shipping addresses.
+When a customer is added to a quotation, the :guilabel:`Invoice Address` and :guilabel:`Delivery
+Address` will autopopulate according to the customer’s contact form.
 
-.. image:: different_addresses/addresses_5.png
+.. image:: different_addresses/quotation-address-autopopulate.png
    :align: center
    :class: img-thumbnail
-   :alt: Automatic quotation fields filling on Odoo Sales
+   :alt: Invoice and Delivery Addresses autopopulate on a quotation
 
-With Odoo Sales, it is now very convenient to play with various addresses in terms of invoice and
-delivery features.
+The :guilabel:`Invoice Address` and :guilabel:`Delivery Address` can also be edited directly from
+the quotation by clicking on the :guilabel:`Internal link` button next to the address line.
```

Comment:
The :guilabel:`Invoice Address` and :guilabel:`Delivery Address` can also be edited directly from
the quotation, by clicking on the :guilabel:`Internal link` button next to the address line.